lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -522,21 +522,25 @@
         description:
             - "If I(true) C(kernel_params), C(initrd_path) and C(kernel_path) will persist in virtual machine configuration,
                if I(False) it will be used for run once."
+            - Parameter is usable with oVirt 4.3 and lower, but it's going to be removed in oVirt 4.4
         type: bool
         version_added: "2.8"
     kernel_path:
         description:
             - Path to a kernel image used to boot the virtual machine.
             - Kernel image must be stored on either the ISO domain or on the host's storage.
+            - Parameter is usable with oVirt 4.3 and lower, but it's going to be removed in oVirt 4.4
```

Comment:
We are trying to eliminate contractions (we're, it's, isn't) to make translating the docs easier. Also, description lines should end with a period. Please change all three lines to:

```suggestion
            - Usable with oVirt 4.3 and lower; will be removed in oVirt 4.4.
```

Reply:
Thank you for change recommendations.